kubernetes/node-watcher/node-watcher-deployment-staging.yaml

```diff
@@ -1,77 +0,0 @@
-apiVersion: apps/v1
```

Comment:
Do we have a way to keep those files around but adapt them with the new variables so that we can deploy things manually as well? it was very handy so far to be able to do it manually for some 1 shot tasks